msteams-platform/concepts/design/plan-to-monetize.md

```diff
@@ -14,7 +14,7 @@ Consider publishing your app on Teams store to offer your app the best platform
 :::row:::
     :::column span="":::
 
-        :::image type="content" source="../../assets/images/app-fundamentals/monetize-apps.png" alt-text="Monetize apps":::
+        :::image type="content" source="../../assets/images/app-fundamentals/monetize-apps.png" alt-text="The screenshot shows the Monetize apps.":::
```

Comment:
It doesn't show monetize apps! And `m` should small - use sentence case.

Reply:
resolved